### ISSUE TITLE
Login in node

### ISSUE THREAD

**Issue by ZeeshanJan** [jeff-zucker/solid-file-client#37]
Hi,

I am giving it a try to login my POD on https://solid.community - it does not logs me in neither throws any error.

```
credentials = {
        "idp"      : "https://solid.community",
        "username" : "peermiles",
        "password" : "", //mypassword
        "base"     : "https://peermiles.solid.community",
        "test"     : "/public/test"
    };
```

and I am trying with both session.webId and as webId.

```
fileClient.login(credentials).then( webId => {
    console.log( `Logged in as ${webId}.`)
}, err => console.log(err) );
```

**Comment by jeff-zucker** [comment]
Sorry about that, it was a bug in the package.  I've fixed it and the new version with the fix is now available on github and npm as version 0.5.1.

**Comment by jeff-zucker** [comment]
The session is what is returned, so here's the right way:
```
fc.login(creds).then( session =>{
  console.log("logged in as "+session.webId)
}, err => { console.log("Login error : "+err); });
```
